index.html

```diff
@@ -4468,14 +4469,14 @@ <h2>The Role of Human-Friendly Identifiers</h2>
 name, a domain name, or a conventional address for a <a>DID controller</a>,
 such as a mobile telephone number, email address, social media username, or
 blog URL. However, the problem of mapping human-friendly identifiers to
-<a>DIDs</a> (and doing so in a way that can be verified and trusted) is
+<a>DIDs</a>, and doing so in a way that can be verified and trusted, is
 outside the scope of this specification.
       </p>
 
       <p>
-Solutions to this problem should be defined in separate specifications that
-reference this specification. It is strongly recommended that such
-specifications carefully consider the:
+Solutions to this problem are defined in separate specifications, such as
+[[?DNS-DID]], that reference this specification. It is strongly recommended that
```

Comment:
Is this `?` still needed? Saw @brentzundel remove it on a different PR.

Reply:
Yes, the question mark means "non-normative" reference and given that DNS-DID isn't an official IETF RFC, we have to make sure we reference it in that way (or it will show up in the normative references section of the specification a trigger an objection from the Director).